pages/common/conda.md

```diff
@@ -0,0 +1,35 @@
+# conda
+
+> Package, dependency and environment management for any programming language.
+
+- Create a new environment, installing named packages into it:
+
+`conda create --name {{environment_name}} {{python=2.7 matplotlib}}`
+
+- List all environments:
+
+`conda info --envs`
+
+- Load or unload an environment:
+
+`source {{activate|deactivate}} {{environment_name}}`
+
+- Delete an environment (remove all packages):
+
+`conda remove --name {{environment_name}} --all`
+
+- Search conda channels for a package by name:
+
+`conda search {{package_name}}`
+
+- Install one or more packages into the current environment:
+
+`conda install {{packages}}`
```

Comment:
@ewels good point -- it would make sense to be consistent here and use the same format as the first example. Maybe replace the second package with something else to better illustrate that they can be switched around.